index.html

```diff
@@ -16,7 +16,7 @@
 	</head>
 	<body>
 		<nav class="prompt">
-
+      <header class="prompt__header js-header"></header>
```

Comment:
Yeah, i notice that github always fucks around with indentation stuff. strange bug.